src/main/resources/openapi/problem.yaml

```diff
@@ -0,0 +1,33 @@
+type: object
```

Comment:
Is this OAS viewable - is this just a template `ProblemError` schema for copying elsewhere?
Wondering what fields are `required`?

Reply:
Intention is to `$ref` this schema in a given package's openapi.yaml. So if I use `audioRequest.yaml` as an example, something like this (these lines would sit under `responses`):

```
        '409':
          description: audio request item already exists
          content:
            application/json-problem:
              schema:
                $ref: "./problem.yaml"
              example:
                type: "AUDIO_REQUEST_100"
                title: "Audio request item already exists"
                status: 409
```

The schema would render as such:
![image](https://github.com/hmcts/darts-api/assets/131763968/25c14f6c-f997-4d0f-9159-1a364f0d9390)

And the example response as:
![image](https://github.com/hmcts/darts-api/assets/131763968/4da5c989-f872-4a23-ab28-54dcf596d18c)

The domain classes would also be generated, although we don't actually need to use them (the intention here is only to document the error responses).

Regarding what is required, good idea I will add `required: true` to the `type`, `title` and `status` fields.

Reply:
Turns out adding `required: true` breaks the build/generator. I think because this constraint is indended for requests rather than responses.